src/hdf5/osw_2ioda.py

```diff
@@ -48,22 +52,15 @@
 meta_keys = [m_item[0] for m_item in MetaDataKeyList]
 
 # The outgoing IODA variables (ObsValues), their units, and assigned constant ObsError.
-obsvars = ['windSpeed']
-obsvars_units = ['m s-1']
-# obserrlist = [1.2]
-obsvars_dtype = ['float']
+obsvars = ['windSpeed', 'windSpeed'+obsErrName, 'windSpeed'+obsQcName]
```

Comment:
Why the other variables are needed?
```suggestion
obsvars = ['windSpeed']
```

Reply:
yes they are needed, this is the data quality indicator particularly for the Spire data where they recommend when the retrieval is valid.   This is the `'windSpeed+obsQcName`

the other is the error estimate `'windSpeed+obsErrName`, this does exist for all the different `osw` we have so far

Reply:
Sorry, @BenjaminRuston. I didn't see where they are used inside the converter. I see `obsvars[0]`, but nothing else. Am I missing something?

Reply:
I see they could maybe used around lines 177-180 in your new version of the code, but they are not used there. This is what makes me think the modification in this line is not needed.

Reply:
yes it's confusing, and yes these are used but these are the names or column titles in the pandas dataFrame, and then when you pull them out to put them into IODA they are extracted again a bit more manually in these two lines:
https://github.com/JCSDA-internal/ioda-converters/blob/9f23fae01decdb7b847d4d596f2b099f056c92e7/src/hdf5/osw_2ioda.py#L179
https://github.com/JCSDA-internal/ioda-converters/blob/9f23fae01decdb7b847d4d596f2b099f056c92e7/src/hdf5/osw_2ioda.py#L180

Reply:
I see it now, it's used only here:

https://github.com/JCSDA-internal/ioda-converters/blob/9f23fae01decdb7b847d4d596f2b099f056c92e7/src/hdf5/osw_2ioda.py#L108